name: read_file_limits
testFormat: workflowfailure

files {
  workflow: read_file_limits/read_file_limits.wdl
  options: read_file_limits/read_file_limits.options
}

metadata {
  workflowName: read_file_limits
  status: Failed
  "calls.read_file_limits.readString.executionStatus": Failed
  "calls.read_file_limits.readString.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readBoolean.executionStatus": Failed
  "calls.read_file_limits.readBoolean.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 7 Bytes."
  "calls.read_file_limits.readLines.executionStatus": Failed
  "calls.read_file_limits.readLines.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readTsv.executionStatus": Failed
  "calls.read_file_limits.readTsv.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readJson.executionStatus": Failed
  "calls.read_file_limits.readJson.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readObject.executionStatus": Failed
  "calls.read_file_limits.readObject.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readObjects.executionStatus": Failed
  "calls.read_file_limits.readObjects.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readMap.executionStatus": Failed
  "calls.read_file_limits.readMap.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 128000 Bytes."
  "calls.read_file_limits.readInt.executionStatus": Failed
  "calls.read_file_limits.readInt.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 19 Bytes."
  "calls.read_file_limits.readFloat.executionStatus": Failed
  "calls.read_file_limits.readFloat.failures.0.causedBy.0.message": ~~"is larger than requested maximum of 50 Bytes."
}
